mpa/det/trainer.py

```diff
@@ -153,11 +154,12 @@ def train_worker(gpu, target_classes, datasets, cfg, distributed=False,
         model = build_detector(cfg.model)
         model.CLASSES = target_classes
         # Do clustering for SSD model
-        if hasattr(cfg.model, 'bbox_head') and hasattr(cfg.model.bbox_head, 'anchor_generator'):
-            if getattr(cfg.model.bbox_head.anchor_generator, 'reclustering_anchors', False):
-                train_cfg = Stage.get_train_data_cfg(cfg)
-                train_dataset = train_cfg.get('ote_dataset', None)
-                cfg, model = cluster_anchors(cfg, train_dataset, model)
+        # TODO[JAEGUK]: Temporal Disable cluster_anchors for SSD model
+        # if hasattr(cfg.model, 'bbox_head') and hasattr(cfg.model.bbox_head, 'anchor_generator'):
+        #     if getattr(cfg.model.bbox_head.anchor_generator, 'reclustering_anchors', False):
+        #         train_cfg = Stage.get_train_data_cfg(cfg)
+        #         train_dataset = train_cfg.get('otx_dataset', None)
+        #         cfg, model = cluster_anchors(cfg, train_dataset, model)
```

Comment:
Is this relevant change?
@jaegukhyun Could you help on this?

Reply:
This one is maybe from push to `otx`, don't know why but will close this PR and this PR moved to #87